src/Database/Auth/ApiKeyAuthenticator.ts

```diff
@@ -101,14 +102,14 @@ export class ApiKeyAuthenticator {
       let publicKey: Buffer, body: IResponseBody;
 
       if (!StatusCodes.isOk(response.statusCode)
-        && !TypeUtil.isObject(body = response.body) && !body.PublicKey
+        && !TypeUtil.isObject(body = response.body as IResponseBody) && !body.PublicKey
```

Comment:
body can be object or array of objects. i cast to specific response type used in this request
also it just or internal use, no additional casts will be needed for library's users